etc/dbus-serialbattery/reinstalllocal.sh

```diff
@@ -3,6 +3,10 @@
 DRIVER=/opt/victronenergy/dbus-serialbattery
 RUN=/opt/victronenergy/service-templates/dbus-serialbattery
 OLD=/opt/victronenergy/service/dbus-serialbattery
+
+#/dev/mmcblk1p3 mountpoint / was made read only in 2.91 release
+mount -o remount,rw /dev/mmcblk1p3 /
```

Comment:
Thanks for sharing your changes. Any reason you did not use the standard Victron remount script?
> /opt/victronenergy/swupdate-scripts/remount-rw.sh